test/e2e/e2e_test.go

```diff
@@ -45,13 +45,13 @@ var _ = ginkgo.Describe("JobSet", func() {
 		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
 
 		// Wait for namespace to exist before proceeding with test.
-		gomega.Eventually(func() bool {
+		gomega.Eventually(func() error {
 			err := k8sClient.Get(ctx, types.NamespacedName{Namespace: ns.Namespace, Name: ns.Name}, ns)
 			if err != nil {
-				return false
+				return err
 			}
-			return true
-		}, timeout, interval).Should(gomega.BeTrue())
+			return nil
+		}, timeout, interval).Should(gomega.Succeed())
```

Comment:
are these changes suggested by the linter?

Reply:
Yea

Reply:
I enabled the linter and I saw there were were a few minor items to fix to get the linter to pass.  

All changes (outside of Makefile) are related to the linter.